fuel-core/src/schema/dap.rs

```diff
@@ -192,4 +192,264 @@ impl DapMutation {
 
         Ok(result)
     }
+
+    #[cfg(not(feature = "debug"))]
+    async fn set_single_stepping(
+        &self,
+        _ctx: &Context<'_>,
+        _id: ID,
+        _enable: bool,
+    ) -> async_graphql::Result<bool> {
+        Err(async_graphql::Error::new(
```

Comment:
Looks like this schema shouldn't be enabled if the `debug` feature isn't on. Since its not the scope of this PR, we could leave an open issue to disable this mod entirely if debug is off.

Reply:
I'll create an issue when this is merged.

Reply:
Since we publish schemas with all features enabled, this is the safest bet.